globals.json

```diff
@@ -557,5 +557,72 @@
 		"YUI": false,
 		"Y": false,
 		"YUI_config": false
+	},
+	"shelljs": {
+		"target": false,
+		"echo": false,
+		"exit": false,
+		"cd": false,
+		"pwd": false,
+		"ls": false,
+		"find": false,
+		"cp": false,
+		"rm": false,
+		"mv": false,
+		"mkdir": false,
+		"test": false,
+		"cat": false,
+		"sed": false,
+		"grep": false,
+		"which": false,
+		"dirs": false,
+		"pushd": false,
+		"popd": false,
+		"env": false,
+		"exec": false,
+		"chmod": false,
+		"config": false,
+		"error": false,
+		"tempdir": false
+	},
+	"prototypejs": {
+		"$": false,
+		"$$": false,
+		"$A": false,
+		"$F": false,
+		"$H": false,
+		"$R": false,
+		"$break": false,
+		"$continue": false,
+		"$w": false,
+		"Abstract": false,
+		"Ajax": false,
+		"Class": false,
+		"Enumerable": false,
+		"Element": false,
+		"Event": false,
+		"Field": false,
+		"Form": false,
+		"Hash": false,
+		"Insertion": false,
+		"ObjectRange": false,
+		"PeriodicalExecuter": false,
+		"Position": false,
+		"Prototype": false,
+		"Selector": false,
+		"Template": false,
+		"Toggle": false,
+		"Try": false,
+		"Autocompleter": false,
+		"Builder": false,
+		"Control": false,
+		"Draggable": false,
+		"Draggables": false,
+		"Droppables": false,
+		"Effect": false,
+		"Sortable": false,
+		"SortableObserver": false,
+		"Sound": false,
+		"Scriptaculous": false
```

Comment:
Does anyone actually use PrototypeJS anymore? I'm pretty sure it's in JSHint only because legacy.